src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/pyproject.toml

```diff
@@ -43,15 +43,15 @@ priority = "primary"
 [tool.poetry.dependencies]
 python = ">=3.8, <3.11"
 ansys-dash-treeview = {version = "0.0.1.dev0", source = "solutions-private-pypi"}
-ansys-saf-glow = {version = "0.4.1", source = "solutions-private-pypi" }
-ansys-saf-portal = {version = "0.5.0", source = "solutions-private-pypi"}
+ansys-saf-portal = {version = "1.0rc3", source = "solutions-private-pypi"}
+ansys-saf-desktop = {version = "1.0rc3", source = "solutions-private-pypi"}
 ansys-solutions-dash-lib = {version = "^0.4", source = "solutions-private-pypi"}
 {% if cookiecutter.with_dash_ui == "yes" %}
-dash = "^2.6"
-dash_bootstrap_components = "^1.2"
-dash-extensions = ">=0.1.13"
-dash-iconify = "^0.1"
-dash-uploader = "^0.6"
+dash = {version = "2.6.1"}
+dash_bootstrap_components = {version = "1.4.1"}
+dash-extensions = {version = "0.1.3"}
```

Comment:
@pierreansys we intentionally constraint dash-extensions to >= 0.1.13. Please keep it.

Reply:
OK. I'll revert the change. 
Cc @JoseRamonRodriguez  since you made this change, in case it was important

Reply:
In no ui mode the developer will need to trigger the dependency resolution again at setup using `-F` option with `setup_environment.py`. The `poetry.lock` is critical for the standard case (SAF+DASH) to ensure the setup will succeed. I think the non-ui mode represents a minor part of the usage right now.